pages/news-and-views/_slug/index.vue

```diff
@@ -1,122 +1,10 @@
 <template>
-	<b-row align-h="center">
-		<div id="container">
+		<Article id="container">
```

Comment:
@TeddTech You aren't actually using the Article component here. It should look like this: https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/master/pages/index.vue#L2